packages/react-components/review/beta/react-components.api.md

```diff
@@ -1253,6 +1253,9 @@ export const _LocalVideoTile: React_2.MemoExoticComponent<(props: {
     personaMinSize?: number | undefined;
 }) => JSX.Element>;
 
+// @beta
+export type LocalVideoTileMode = 'mobile' | 'desktop' | 'hidden' | 'responsive';
```

Comment:
for hidden, can we just make the localParticipant optional? Or do other video gallery things rely on that?

Reply:
unfortunately we do provide the local participant to get their screen share. there is a case that Contoso could provide a audio only experience but still allow a screen share stream so we cannot just remove the participant.